Makefile

```diff
@@ -101,6 +101,11 @@ fmt: ## Run go fmt against code.
 	@echo "Running go fmt"
 	go fmt ./...
 
+.PHONY: golangci-lint
+golangci-lint: ## Run golangci-int against code.
+	@echo "Running golangci-lint"
+	HOME=/tmp hack/golangci-lint.sh
```

Comment:
There's already a hack/lint.sh and Make target. Any reason to not just add golangci-lint to that existing script? That script could just become the "general purpose" linter tool, possibly growing in the future to include other linters like pylint, shellcheck, bashate, etc.

Reply:
Well, basically I put that out there in order to have a small script per linter. i.e. golint allows you to have the `-set-exit-status param`which is used in that bash script but it wouldn't apply to golangci-lint nor to anyone else. What we could do, though, is to aggregate such scrips calls into one while keeping them separate into files.